demos/noise_suppression_demo/python/noise_suppression_demo.py

```diff
@@ -30,20 +19,21 @@
 log.basicConfig(format='[ %(levelname)s ] %(message)s', level=log.DEBUG, stream=sys.stdout)
 
 
-def build_argparser():
-    parser = ArgumentParser(add_help=False)
-    args = parser.add_argument_group('Options')
-    args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
-    args.add_argument("-m", "--model", help="Required. Path to an .xml file with a trained model",
-                      required=True, type=Path)
-    args.add_argument("-i", "--input", help="Required. Path to a 16kHz wav file with speech+noise",
-                      required=True, type=str)
-    args.add_argument("-o", "--output", help="Optional. Path to output wav file for cleaned speech",
-                      required=False, type=str, default="noise_suppression_demo_out.wav")
-    args.add_argument("-d", "--device",
-                      help="Optional. Target device to perform inference on. "
-                           "Default value is CPU",
-                      default="CPU", type=str)
+def build():
+    parser = ArgumentParser()
+    parser.add_argument("-m", "--model", required=True, type=Path, metavar="<MODEL FILE>",
+        help="path to an .xml file with a trained model")
+
+    parser.add_argument("-i", "--input", required=True, type=Path, metavar="<WAV>",
+        help="path to an input 16kHz WAV file")
+
+    parser.add_argument("-d", "--device", default="CPU", metavar="<DEVICE>",
+        help="specify a device to infer on (the list of available devices is shown below). Default is CPU")
+
+    parser.add_argument("-o", "-output", default="noise_suppression_demo_out.wav", metavar="<WAV>",
+        help="path to an output WAV file. Default is noise_suppression_demo_out.wav")
+    log.info('OpenVINO Runtime')
+    log.info('\tbuild: {}'.format(get_version()))
```

Comment:
I would prefer to distinguish parser building and logging. For Python Model API demos the logging depends on adapter

Reply:
Updated